app/data_providers/slate_providers/topic_slate_provider.py

```diff
@@ -49,7 +49,13 @@ def headline(self) -> str:
 
     @property
     def more_link(self) -> Optional[LinkModel]:
-        return LinkModel(text=f'Explore more {self.topic.name}', url=f'https://getpocket.com/explore/{self.topic.slug}')
+        # Topic pages only exist for en-US.
+        if self.locale == LocaleModel.en_US:
+            return LinkModel(
+                text=f'Explore more {self.topic.name}',
```

Comment:
I see that there's no support for German, but anyway I think all strings should be in one place for consistency, meaning json files.

Reply:
Agreed. I kept this out-of-scope because I prefer smaller PRs.

Translating topics would partly follow a similar approach to other headlines, but would require injecting translations into the topics model.